### ISSUE TITLE
[SUGGESTION] Calcium

### ISSUE THREAD

**Issue by Ultima1990** [pokeclicker/pokeclicker#2310]
TLDR: Reduce Proteins per region, introduce Calcium as a new additional vitamin that scales with steps usable equally as many times as Protein per region, total vitamins should hang around the old 5 number so it'll be either 4 or 6.

Originally Protein was going to be a multiplier or % boost based on attack. It was turned into a flat equal +1 boost for everyone because we were afraid of the already strong mons snowballing. Since, obviously, if we do a multiplier or % boost someone with big attack gets even bigger attack. Looking at you, Gyarados.

Sadly, the desired result was not achieved. Gyarados still snowballs. But not because of his high attack. It's because of his low steps. Low step mons benefit significantly more from Protein.

So here's a pitch for a new vitamin. I picked Calcium for this because in the main series Calcium boost special attack, and our Calcium is also going to boost attack gains, but in a sort of "special" way. It's a boost that scales with required steps. If you need more steps to hatch, you get a bigger boost. Less steps, less boost. It should probably scale hard. From like a 0.1 boost at 200 steps to a 5 boost at 4800 steps (placeholder numbers just to give an idea). The goal here is to make inefficient mons more efficient, without necessarily making already efficient mons less efficient. Yes, Magikarp will become relatively less efficient because other mons are getting a bigger boost than it, but Magikarp's actual efficiency number will not go down because of this.

Except maybe Magikarp's efficiency number will go down. Because as an addition to this _that's up for debate_ I think we should reduce max Protein per region, in order to avoid making mons too OP with too many vitamins. Instead of 5 Protein and 5 Calcium per region, which is 10 total vitamins per region, which I think is too much, we'd do either 2 of both or 3 of both, ending with 4 or 6 total vitamins per region. I'd like to keep the vitamins equal in this regard. So the player would either go up one vitamin per region, or down one vitamin per region. Can't really feed half a vitamin to stay at 5.
Yes, this would mean retroactively removing already used Proteins over the cap from your mons. No, I do not intend to give a money refund for them, you already got the benefit of breeding OP 320 step Gyarados with 35 Proteins which newer players will not get, I think that's compensation enough.

No, I do not intend do just do a 5 vitamins per region cap and let the player choose which vitamins to fill that cap with. Any smart player would still choose Protein for Gyarados and that is the exact mon that needs to be taken down a notch. Give that boi some Calcium, it needs a balanced diet. Equal vitamins!

**Comment by jk13pclick** [comment]
Why don't we just change the Protein Attack Boost to scale based on Egg Steps? Why introduce a new Vitamin? Example Desmos graph attached for simple balancing of attack boost around egg steps: https://www.desmos.com/calculator/nuo4c0i2dc

**Comment by Ultima1990** [comment]
Because I don't want to fully take away that nice Protein boost. Tis a good strong boost. Players like good strong boost. Players dislike fully losing good strong boost. So we just reduce good strong boost and replace it with different boost instead of scrapping it entirely.

**Comment by jk13pclick** [comment]
Just dropping this here: Example Calcium Bonus Desmos: https://www.desmos.com/calculator/bnsf7aaazh

**Comment by Aegyo** [comment]
Closing, two new vitamin choices were added in #3473
